src/jobs/integration-test.yaml

```
@@ -0,0 +1,43 @@
+description: |
+  Runs an integration test by creating a KIND cluster and executing it as a Go
+  test.
+
+  Uses the machine executor and replaces the Go version with a version
+  controlled by us since the preinstalled Go is very outdated.
+
+  Tests can be customized by providing a KIND config file or a setup script
+  which is run before the test is executed.
+environment:
+  GOPATH: /go
+  KUBERNETES_VERSION: v1.16.3
+executor: machine
+parameters:
+  env-file:
+    default: ".env"
```

I guess empty is a sane default here. We don't need to always have env file.

```suggestion
    default: ""
```